app/services/user.ts

```diff
@@ -223,6 +226,8 @@ export class UserService extends PersistentStatefulService<IUserServiceState> {
     await this.sceneCollectionsService.safeSync();
     // signs out of chatbot
     await this.chatbotApiService.logOut();
+    // clear available features
+    await this.incrementalRolloutService.clearAvailableFeatures();
```

Comment:
What is the goal of using PersistanceStatefulService if you always call `clearAvailableFeatures`?